### DISCUSSION TITLE
[Mend Renovate app] What are the definitions of the different `renovateStatus` values?

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#29728] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I'm trying to work out what's different between `onboarded` and `activated` definitions in the Mend Renovate app

I can guess what `onboarding` (there's an onboarding PR raised) and `disabled` (the repo no longer has a `renovate.json` / the app is disabled) are, but unclear on the other two.

**Comment by rarkins** [comment, marked as answer]
onboarded = onboarding PR merged - or no config necessary
activated = onboarded _and_ has merged at least one update PR

**Reply by jamietanna** [reply]
Thanks Rhys!

**Comment by mikaello** [comment]
Just for future reference, there is some more information about this in mend-ce-ee docs: https://github.com/mend/renovate-ce-ee/blob/main/docs/configuration-options.md#job-scheduling-options

> Note: An activated repository is one that has onboarded and also accepted at least one Renovate PR.
